README.md

```diff
@@ -272,6 +272,10 @@ This library tries to minimize the amount of "special handling" that a cross-bro
 
 This section aims to keep track of the most common issues that an extension may have.
 
+### No callback supported by the Promise-based APIs on Chrome
+
+While Firefox some of the asynchronous API methods (the ones that return a promise) also support the callback parameter (mostly as a side effect of the backward compatibility with the callback-based APIs available on Chrome), the Promise-based APIs provided by this library do not support the callback parameter (See ["#102 Cannot call browser.storage.local.get with callback"][I-102]).
```

Comment:
I think that you accidentally dropped an "in" here. I suggest to reword the start from

> While Firefox some of the asynchronous API methods (...) also support

to

> While some of the asynchronous API methods in Firefox